next/src/pages/AboutUs.tsx

```diff
@@ -1,64 +1,88 @@
-import React from "react";
+import React , { useState, useEffect } from "react";
+
+import styled from "styled-components";
+
+import { NavbarOpenHandler } from "../components/navbar/types";
+import Navbar from "../components/navbar/Navbar";
+import Footer from "../components/footer/Footer";
+
+const PageContainer = styled.div`
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+`;
 
 export default function AboutUs() {
+
+  const [navbarOpen, setNavbarOpen] = useState(false);
+
+	const handleToggle: NavbarOpenHandler = () => {
+		setNavbarOpen(!navbarOpen);
+	};
+
   return (
-    <div>
-      <h1>About</h1>
-      <p>
-        CSESoc is the official representative body of computing students at
-        UNSW. We are one of the largest and most active societies at UNSW, and
-        the largest computing society in the southern hemisphere. CSESoc
-        comprises ~9,500 UNSW students spanning across degrees in Computer
-        Science, Software Engineering, Bioinformatics and Computer Engineering.
-        We are here to fulfil the social, personal and professional needs of CSE
-        students, and promote computing through a variety of forms.
-      </p>
-      <p>
-        We are a society for the students, by the students. Here’s an overview
-        of what we do;
-      </p>
-      <ul>
-        <li>
-          Run weekly social and educational events, including trivia, movie,
-          boardgames nights, LAN parties, workshops, coding competitions, tech
-          talks, and our famous free weekly BBQ.
-        </li>
-        <li>
-          Create original media content, including Podcasts, articles, YouTube
-          videos, and live streams
-        </li>
-        <li>
-          Run a highly successful First Year Camp and Peer Mentoring program,
-          offering new CSE students (both undergraduate and postgraduate) a
-          chance to meet and mingle with other newcomers
-        </li>
-        <li>
-          Engage students with industry sponsors and representatives to develop
-          their professional capacity and curiosity
-        </li>
-        <li>
-          Develop our own open-source projects for students to get learn new
-          skills and develop tools for our community
-        </li>
-        <li>
-          Facilitate an online community of ~3k Discord users, ~5k Facebook
-          followers, ~600 YouTube subs, and ~500 Instagram followers
-        </li>
-      </ul>
+    <PageContainer>
+      {!navbarOpen && <Navbar open={navbarOpen} setNavbarOpen={handleToggle} />}
+
+      <div style={{ paddingLeft: "80px", paddingRight:"80px", paddingTop:"180px", paddingBottom: "180px" }}>
```

Comment:
consider changing this to use styled components instead of inline styling,
furthermore, look into mobile compatibility and whether the padding will affect the text on a smaller screen